monitor.py

```diff
@@ -122,6 +122,9 @@ def initial_start_time(self):
             return self.default_start_time
         return datetime.datetime.utcnow()
 
+    @property
+    def message_prefix(self):
+        return '[{}] '.format(self.collection.name) if self.collection else ''
```

Comment:
Since a `Monitor` is associated with a `collection`, I think this would go more easily into the name of the logger (set in `Monitor.log`). That would get you the result you want with fewer code changes.

Reply:
I'll dig deeper into the logger approach. I started to go down that path, but was in a hurry. But I think it is a better way to go, so I'll find a little more time for this. Thanks.